app/helpers/search_helper.rb

```diff
@@ -85,6 +85,20 @@ def ordering
     end
   end
 
+  # Transforms controller params from get request into query args for a search
+  # @return [Hash]
+  def get_query_args
```

Comment:
Nested objects cannot be passed in a query string, here I am proposing a standard method for passing aggregator filters in a query string, that could help with some of the other tickets as well.

Instead of passing the parameters in the body like this
```
{
  "agg_filters": {"city": {"Beijing": 1}}
}
```
They are being passed in a query string as `agg_filters[]=city&city[]=Beijing`, so that query params look like this:
```
{
  "agg_filters": ["city"],
  "city": ["Beijing"]
}
```
This format can then easily be transformed to the one that query_args is expecting